lib/SILOptimizer/Analysis/EscapeAnalysis.cpp

```diff
@@ -1070,6 +1070,11 @@ static bool mayContainReference(SILType Ty, const SILFunction &F) {
 
 bool EscapeAnalysis::isPointer(ValueBase *V) {
   auto *F = V->getFunction();
+
+  // The function can be null, e.g. if V is an undef.
+  if (!F)
```

Comment:
Is it possible for this to happen for anything but undef? Maybe put in an assert?

Reply:
I think currently not. This is the safe default (in case we add another SILValue kind in the future)

Reply:
Yes, but we won't know to update this code when someone adds something new. They won't be able to know to make the decision. With the assert, when someone adds the new SILValue, they get a hard warning that they may need to update this code.